crates/sui-faucet/src/metrics_layer.rs

```diff
@@ -75,10 +75,15 @@ where
         let future = Box::pin(async move {
             let resp = inner.oneshot(req).await;
             match &resp {
-                Result::Ok(resp) if !resp.status().is_success() => metrics.failed(),
+                Result::Ok(resp) if !resp.status().is_success() => metrics.failed(None),
```

Comment:
Less familiar with the metrics code so not sure if it's possible or how easy it might be, but is there any way of attaching the status here as well?

Reply:
Good point, I believe that would be easy for this case, let me see